frontend/src/lib/components/PropertyKeyInfo.tsx

```diff
@@ -154,6 +159,10 @@ export const keyMapping: KeyMappingInterface = {
             description: 'Keys of the feature flags that were active while this event was sent.',
             examples: ['beta-feature'],
         },
+        $enabled_feature_flags: {
```

Comment:
Quick flyby to remove this :)  See https://github.com/PostHog/posthog-js-lite/issues/13